torchaudio/transforms.py

```diff
@@ -507,18 +409,7 @@ def __call__(self, x):
             x_mu (LongTensor or ndarray)
 
         """
-        mu = self.qc - 1.
-        if isinstance(x, np.ndarray):
```

Comment:
In general we don't want to support this. If Someone wants to use numpy, they'll need to use our 0-cost conversions from their numpy array to a tensor, then pass it into this and then convert it back. We also don't want to use numpy to do our computations.